mula/.ci/.env.test

```diff
@@ -5,13 +5,13 @@ POSTGRES_DB=scheduler
 SCHEDULER_API_HOST="0.0.0.0"
 SCHEDULER_API_PORT="8000"
 SCHEDULER_PQ_MAXSIZE=0
-SCHEDULER_PQ_INTERVAL=60
-SCHEDULER_PQ_GRACE=1
-SCHEDULER_RABBITMQ_DSN=amqp://guest:guest@ci_rabbitmq:5672/%2fkat
+SCHEDULER_PQ_POPULATE_INTERVAL=60
+SCHEDULER_PQ_POPULATE_GRACE_PERIOD=1
+QUEUE_URI=amqp://guest:guest@ci_rabbitmq:5672/%2fkat
```

Comment:
might be a bit confusing naming this `QUEUE_URI` since it mainly directs to the rabbitmq host 'messaging queue' from which the scheduler pulls messages from. It could be misinterpreted that this might be the queue of the scheduler itself.